src/components/checkboxTree/types.ts

```diff
@@ -0,0 +1,14 @@
+export type CheckboxTreeProps = {
+  composition: Composition[];
+  preSelected?: string[];
```

Comment:
maybe just `selected` and `expanded` would be more accurate, what do you think?

Reply:
Selected and expanded are used inside the component itself, renaming them like this will result in variable name overlapping.

Reply:
yeah, but the names are misleading, because they represent the current state of the checkbox tree, not the "pre" (initial) values

Reply:
Changed preExpanded to expanded and preSelected to checked, is this good?